depz/depz.py

```diff
@@ -1210,6 +1210,14 @@ def do_show_outdated (self, rname, rr):
     git = Git(rr.val.full_directory)
 
     current_branch = git.current_branch
+    if current_branch is None:
+      llog.warn("No current branch or in a detached HEAD state. ")
+      if "checkout" in rr._dict:
+        llog.warn("Manually 'git checkout " + rr.val.checkout + "' in the repo.")
+      else:
+        llog.warn("Manually 'git checkout master' (or other default branch) in the repo.")
+      return
+
```

Comment:
Two thoughts:
1) I think maybe they should be more clearly expressed as suggestions.  If there was something *clearly correct* to do here, depz would just do it.  So I think the phrasing should clearly express this.  Like, "You may want to switch to the checkout branch ('git checkout foo')." as one example.  Or "Maybe you want to switch to the default checkout branch (git checkout foo)?"
2) Similarly, I think maybe we should offset these suggestion type log messages from others.  Specifically, I suggest: 2a) They are done at INFO instead of WARN, and 2b) The messages are in parentheses.

I'm not super married to 2b in particular.  Thoughts?

Reply:
2a: the downside of logging the suggestions at the INFO level is that the tips don't line up with the original warning anymore, columnwise:
```
~/Projects/Games/light-before-xmas> depz --ff
WARNING:depz.Plasmacore:No current branch or in a detached HEAD state.
INFO:depz.Plasmacore:(You may want to manually 'git checkout master' in the repo.)
```
IMO it's better as a WARNING for that reason. Your call. What think?

Reply:
Alternatively we could change WARNING to print as WARN instead...

Reply:
If you have coloredlogs installed (the README discusses it), the logs are much easier to read in general, and this fixes the alignment of messages.  That is, this is highly recommended to begin with.

I think we should just fix the default format string to use a fixed-sized field for the level name.  Then we can log at INFO and things stay aligned even without coloredlogs.  I've opened this as #8.

Reply:
Just changing WARNING to WARN doesn't fix ERROR and DEBUG and other levels, though they're not as important.  We could probably change ERROR to ERR and CRITICAL to CRIT, etc., so everything looked good with only four spaces, though we'd probably have to monkey patch to do it?  Or write our own log formatter class?

On the other hand, my intention for #8 is to just change the default log format string to like "%(levelname)-8s: %(message)s" or something like that.  All the level names fit in 8 characters, I think, and everything would align.